geode-core/src/test/java/org/apache/geode/test/dunit/standalone/ProcessManager.java

```diff
@@ -90,12 +92,17 @@ public synchronized void launchVM(String version, int vmNum, boolean bouncedVM)
     }
 
     // TODO - delete directory contents, preferably with commons io FileUtils
-    Process process = Runtime.getRuntime().exec(cmd, null, workingDir);
-    pendingVMs++;
-    ProcessHolder holder = new ProcessHolder(process);
-    processes.put(vmNum, holder);
-    linkStreams(version, vmNum, holder, process.getErrorStream(), System.err);
-    linkStreams(version, vmNum, holder, process.getInputStream(), System.out);
+    try {
+      Process process = Runtime.getRuntime().exec(cmd, null, workingDir);
+      pendingVMs++;
+      ProcessHolder holder = new ProcessHolder(process);
+      processes.put(vmNum, holder);
+      linkStreams(version, vmNum, holder, process.getErrorStream(), System.err);
+      linkStreams(version, vmNum, holder, process.getInputStream(), System.out);
+    } catch (RuntimeException | Error t) {
+      t.printStackTrace();
+      throw t;
+    }
```

Comment:
Worth logging a message telling us where it is printed from with the stack trace?